newIDE/app/src/ObjectsRendering/Renderers/RenderedUnknownInstance.js

```diff
@@ -39,12 +39,8 @@ export default class RenderedUnknownInstance extends RenderedInstance {
   }
 
   update() {
-    const width = this._instance.hasCustomSize()
-      ? this._instance.getCustomWidth()
-      : 32;
```

Comment:
we're losing this 32 in the refactor, I don't think it's a problem but double checking you're aware of that

Reply:
`32` is value returned by the default implementation.